exetera/core/dataset.py

```diff
@@ -1,207 +1,176 @@
-# Copyright 2020 KCL-BMEIS - King's College London
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#     http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
 
-import csv
-import time
-import numpy as np
+class Dataset():
```

Comment:
Can you put the license boilerplate here?

Reply:
License info pushed in commit 7cfeceb